mrmap/resourceNew/models/service.py

```diff
@@ -149,6 +149,12 @@ def get_harvest_url(self) -> str:
                 pass
         return ""
 
+    def get_validate_url(self) -> str:
```

Comment:
i think it would be better to use Conformity Check Run.get_add_url() instead of implementing a code duplication which depends on other apps here.

Reply:
See above. I am not sure that I get the idea completely.

Reply:
see answer above.